bin/import.ml

```diff
@@ -29,6 +29,8 @@ module Profile = Dune_rules.Profile
 module Log = Dune_util.Log
 include Common.Let_syntax
 
+let in_group (t, info) = (Term.Group.Term t, info)
```

Comment:
This name is not super clear to me. I feel like if we were writing this code from scratch, we'd write `Term t` for each command definition. And so, this function only exist because of the way the code has evolved over the years.

In general, it is better to keep the code looking the same as what we would write from scratch. So here we should push this `Term ...` in all command definitions.

Reply:
Yes, that's a good point, but I intentionally wrote it this way to minimize the diff. My reasoning is that Daniel might want to tweak the api, so it's better to delay changing our entire codebase until the api is settled.

Reply:
Makes sense. We should also wait until the design is settled on the cmdliner side before merging then, to avoid having the behaviour changing (I remember that they were some discussions around the presentation of the help). Could you put WIP in the title?

Reply:
@jeremiedimino i think we should reconsider merging this PR. It's currently blocking working on dune rpc, and it would also be useful in other dune 3 related clean ups. Daniel already what kind of design he wants, and this PR implements exactly that. I'm just waiting for code review of the implementation at this point. Whatever changes to the api he makes are going to be trivial for me to adopt anyway. 

So let's just merge this now and tweak this again once upstream is updated.

Reply:
SGTM!